src/main/com/diana/restaurant/enums/Queues.java

```diff
@@ -0,0 +1,5 @@
+package com.diana.restaurant.enums;
+
+public final class Queues {
```

Comment:
What do you think about using enums to define queues?

Reply:
I don't see how it will be different, since all queues will be managed in the QueueManager 

I've investigated about enums being type-safe which might be useful for working with threads, it might also be more readable to work using enums in this case